tests/show.rs

```diff
@@ -36,6 +36,7 @@ fn fuelup_show_latest() -> Result<()> {
                 - forc-client
                   - forc-deploy : 0.1.0
                   - forc-run : 0.1.0
+                  - forc-submit : not found
```

Comment:
Shouldn't these show 0.1.0 as well?

Reply:
Doh! Oh wow, ok I see where I went wrong... I assumed that file didn't exist because it was pulling down an old version of from githubusercontent.com where `forc-submit` doesn't exist, but they're actually coming from mocked files defined by `ALL_BINS`. I'll fix that, thanks!